paper-ieee-trans.tex

```diff
@@ -90,17 +87,17 @@ \section{Introduction}
 
 Analysing the EC--funded software projects over the last decade,
 one can observe a continuous increase in the prominence and robustness of software
-testing and validation procedures \cite{aiftimiei}. The
-evolution of the software engineering methodologies (creation of the Agile
+testing and validation procedures \cite{aiftimiei}. This tendency was motivated by the
```

Comment:
-1

Reply:
+1 as the previous comment for clearness